### ISSUE TITLE
Feature: Not a JSON object

### ISSUE THREAD

**Issue by ianmeinert** [TechReborn/TechReborn#2210]
**Describe the bug**
JSON objects are failing to load on world start up.

**Steps to Reproduce**
1. install TechReborn
2. generate world
3. watch the logs.

1. Rejoin the same world
2. watch the logs

**Screenshots**
n/a

**Environment (please complete the following information with the version):**
 - Minecraft: 1.16.2
 - Mod Loader: fabric 0.9.3+build.207
 - TechReborn-1.16-3.5.2+build.110

**Logs**
Pastebin the log [e.g. `logs/latest.log`] and link it below:
`[17:40:32] [Netty Client IO #8/ERROR]: Feature: Not a JSON object: "techreborn:ore_techreborn_tungsten_ore"; Not a JSON object: "techreborn:ore_techreborn_sodalite_ore"; Not a JSON object: "techreborn:ore_techreborn_sheldonite_ore"; Not a JSON object: "techreborn:ore_techreborn_peridot_ore";`

**Additional context**
Affects single and multiplayer

**Comment by ghost** [comment]
If you have KubeJS, can you try with its latest version?
If this doesn't solve it, can you provide your entire log?

**Comment by ianmeinert** [comment]
We aren't using KubeJS... yet.

Pastebin: https://pastebin.com/QecQ3gWU

See timehack [21:33:52]

**Comment by ghost** [comment]
Does the error `Not a JSON object: "minecraft:enderite_ore_feature"` also go away when you load your instance without TechReborn and RebornCore?

What happens with the latest fabric API?
https://www.curseforge.com/minecraft/mc-mods/fabric-api/files/3049174

I'm not certain this is an actual TR issue or something else.  Hopefully a dev can chime in.

**Comment by ianmeinert** [comment]
The `Not a JSON object: "minecraft:enderite_ore_feature"` is an [issue](https://github.com/Nic4Las/Minecraft-Enderite-Mod/issues/13) with another mod. I removed TR and the `Not a JSON object: "minecraft:enderite_ore_feature"` remained.

I upgraded to the latest 20.1 build 401, which resulted in the same errors.

The other issue points to a class that implements Minecraft's BuiltinRegistries. In TechReborn's case, it would be the WorldGenerator.java, line 125. This may be tied to [MC-196068](https://bugs.mojang.com/browse/MC-196068) which identifies an issue with a feature of a feature.
